Package.swift

```diff
@@ -28,14 +28,14 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/ml-explore/mlx-swift", .upToNextMinor(from: "0.21.2")),
-        .package(url: "https://github.com/ml-explore/mlx-swift-examples", exact: "1.18.1"),  // Pin MLX Swift Examples as it doesn't follow semantic versioning
+        .package(url: "https://github.com/ml-explore/mlx-swift-examples", exact: "2.21.2"),  // Pin MLX Swift Examples as it doesn't follow semantic versioning
         .package(url: "https://github.com/huggingface/swift-transformers", .upToNextMinor(from: "0.1.14")),
         .package(url: "https://github.com/StanfordBDHG/OpenAI", .upToNextMinor(from: "0.2.9")),
         .package(url: "https://github.com/StanfordSpezi/Spezi", from: "1.2.1"),
         .package(url: "https://github.com/StanfordSpezi/SpeziFoundation", from: "2.0.0"),
         .package(url: "https://github.com/StanfordSpezi/SpeziStorage", from: "1.0.2"),
         .package(url: "https://github.com/StanfordSpezi/SpeziOnboarding", from: "1.1.1"),
-        .package(url: "https://github.com/StanfordSpezi/SpeziChat", .upToNextMinor(from: "0.2.1")),
+        .package(url: "https://github.com/StanfordSpezi/SpeziChat", exact: "0.2.1"),
```

Comment:
What's the reason for not keeping it as `upToNextMinor` and even bumping it to `0.2.2`?

Reply:
@PSchmiedmayer this is fixed to 0.2.1, because using 0.2.2 leads to the following error when building the TestApp:

Non-sendable type 'SFSpeechRecognitionResult?' returned by implicitly asynchronous call to nonisolated function cannot cross actor boundary

@LeonNissen and I both independently encountered this. Unclear where the issue is coming from, but it's likely unrelated to SpeziLLM.

Reply:
@PSchmiedmayer this is fixed to 0.2.1, because using 0.2.2 leads to the following error when building the TestApp, which originates from line 212 of MessageInputView.swift within SpeziChat:

Non-sendable type 'SFSpeechRecognitionResult?' returned by implicitly asynchronous call to nonisolated function cannot cross actor boundary

@LeonNissen and I both independently encountered this.

Reply:
@PSchmiedmayer this is fixed to 0.2.1, because using 0.2.2 leads to the following error when building the TestApp, which originates from line 212 of MessageInputView.swift within SpeziChat:

Non-sendable type 'SFSpeechRecognitionResult?' returned by implicitly asynchronous call to nonisolated function cannot cross actor boundary

@LeonNissen and I both independently encountered this.

Reply:
The `Sendable` warning should come almost certainly from SpeziSpeech. Interesting that this doesn't happen in 0.2.1 but does happen in 0.2.2?

Reply:
Oh I see, 0.2.2 enables Swift 6 language mode for SpeziChat and SpeziSpeech, that's why the error is surfaced here then.

Reply:
It is very weird that this issue occurs now, probably an Xcode 16.2 thing. Back with 16.1 this wasn't an issue.

Reply:
Should we just fix it to 0.2.1 so we can merge this, and then solve this issue later?

Reply:
Never mind, I see your PR https://github.com/StanfordSpezi/SpeziSpeech/pull/11. Thanks!

Reply:
Mhh this is a super weird issue. For me, SpeziChat resolves its SpeziSpeech dependency to version 1.2.0, which doesn't even exist, see here: https://github.com/StanfordSpezi/SpeziSpeech/tags
The 1.2.0 "version" corresponds to the 1.1.0 code, not 1.1.1. I'm super confused, and I cannot really tell you why that is happening.
The PR you mentioned (https://github.com/StanfordSpezi/SpeziSpeech/pull/11) doesn't fix that problem, the problem is already fixed in version 1.1.1.

Reply:
@vishnuravi Can you check which SpeziSpeech version is checked out with the SpeziLLM UI test app? Is it 1.2.0?

Reply:
Yes, it is resolved to SpeziSpeech 1.2.0.

Reply:
<img width="286" alt="image" src="https://github.com/user-attachments/assets/5d941d03-f87b-4d49-ab25-c7bc7b319260" />

Yep same for me. I have no idea how that happens, there isn't a 1.2.0 release OR 1.2.0 tag for the SpeziSpeech repo:  https://github.com/StanfordSpezi/SpeziSpeech/tags
The code state that is checked out with the "1.2.0 version" is quite old, hence the `Sendable` errors.

@PSchmiedmayer Any ideas here?

Reply:
Just verified the list of tags, there isn't a 1.2.0:

<img width="852" alt="image" src="https://github.com/user-attachments/assets/d8e55561-7220-4bcc-bdad-e74f2d00a495" />

Reply:
We tagged new versions in Speech and Chat; hopefully this resolves the moved tags. Really strange where this came from.

Reply:
This looks to be solved, thank you @PSchmiedmayer and @philippzagar!